app/assets/javascripts/admin/addon/models/admin-dashboard.js

```diff
@@ -42,6 +43,25 @@ AdminDashboard.reopenClass({
       return model;
     });
   },
+
+  fetchNewFeatures() {
+    return ajax("/admin/dashboard/new-features.json").then((json) => {
+      const model = AdminDashboard.create();
```

Comment:
It seems a little confusing to me to re-use the `AdminDashboard` model for a different kind of data. That means two instances of `AdminDashboard` might be representing very different things. 

This model seems very simple. It wouldn't be so bad to have your component make the ajax call and set the properties on itself without wrapping them in an `EmberObject` even. I doubt they are observed.

Another slightly more complex but flexible approach would be to use the store. Change the component to do `this.store.find('new-feature')` - you'll need an Adapter and perhaps to slightly modify your serializer.